pkg/server/container_create.go

```diff
@@ -192,9 +192,12 @@ func (c *criService) CreateContainer(ctx context.Context, r *runtime.CreateConta
 	meta.ImageRef = image.ID
 	meta.StopSignal = image.ImageSpec.Config.StopSignal
 
-	// Get container log path.
-	if config.GetLogPath() != "" {
+	// Validate log paths and compose full container log path.
+	if sandboxConfig.GetLogDirectory() != "" && config.GetLogPath() != "" {
 		meta.LogPath = filepath.Join(sandboxConfig.GetLogDirectory(), config.GetLogPath())
+		logrus.Debugf("Composed container full log path %q using sandbox log dir %q and container log path %q", meta.LogPath, sandboxConfig.GetLogDirectory(), config.GetLogPath())
+	} else {
+		logrus.Debugf("Empty log path in either sandbox or container config. Container full log path: %q, sandbox log dir: %q, container log path: %q", meta.LogPath, sandboxConfig.GetLogDirectory(), config.GetLogPath())
```

Comment:
nit can we move this one to warning or error level...

Reply:
1) No need to print the paths, because they are empty.
2) I think a lot of crictl examples don't have log path specified, I think an info might be good enough.

Reply:
1. Well there is at least one of them is empty but probably not both. So I will still print the paths for easy debugging.
2. Yeah I think info is OK.